sqlness-cli/src/main.rs

```diff
@@ -65,6 +66,21 @@ impl EnvController for CliController {
     async fn stop(&self, _env: &str, _db: Self::DB) {}
 }
 
+struct PostgresController {
+    db_config: DatabaseConfig,
+}
+
+#[async_trait]
+impl EnvController for PostgresController {
```

Comment:
I think add PostgresController is a valid option, and you need to also update L80 to something like this:
```
let ctrl = match type {
mysql => ...
postgres => ...
}
```